src/Router.js

```diff
@@ -1,14 +1,22 @@
 import React from 'react';
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
-import KimCodeLogin from './pages/KimCode/Login/Login';
-import KimCodeMain from './pages/KimCode/Main/Main';
+// import KimCodeLogin from './pages/KimCode/Login/Login';
+// import KimCodeMain from './pages/KimCode/Main/Main';
+
+import Login from './pages/Hwangsoohyeon/Login/Login';
+import Join from './pages/Hwangsoohyeon/Join/Join';
+import Join_ok from './pages/Hwangsoohyeon/Join/Join_ok';
```

Comment:
컴포넌트 이름은 대문자로 시작하고 camelCase로 작성하는것 지켜주세요~
```suggestion
import JoinOk from './pages/Hwangsoohyeon/Join/JoinOk';
```